practice-one/src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Home = () => {
+  return (
+    <>
```

Comment:
Sorry I forgot to remove it, because before I added an html tag to test it, I needed a div or a fragment to wrap it.
Now I deleted that fragment
![image](https://github.com/datdragon777/React-Training/assets/74806826/3d2c8f3a-78bf-4747-8d47-94fce2b70772)
![image](https://github.com/datdragon777/React-Training/assets/74806826/8860982e-982e-4907-bce5-e321f3b6fe10)